forwards_paper.tex

```diff
@@ -332,12 +348,16 @@ \subsection*{Recording the pedigree in forwards time}
 \algstep{W5.}{New node.}{Set $P'_j \leftarrow n$ and $\tau_n \leftarrow t$.
 }
 
-\algstep{W6.}{Choose parents.}{Set $a \leftarrow \randomuniform(\{0, \dots, N - 1\})$,
-    $b \leftarrow \randomuniform(\{0, \dots, N - 1\})$ and $x \leftarrow \randomuniform([0, L))$.
+\algstep{W6.}{Choose parents.}{Set $a \leftarrow \randomuniform(\{0, \dots, n - 1\})$,
+    $b \leftarrow \randomuniform(\{0, \dots, n - 1\})$ and $x \leftarrow \randomuniform([0, L))$.
+}
+
+\algstep{W7.}{Record edges}{Set $E \leftarrow E \cup \left\{ (0, x, P_a, n), (x,
+		L, P_b, n) \right\}$.
 }
 
-\algstep{W7.}{Record edges}{Set $E \leftarrow E \cup \left\{ (0, x, P_a, n), (x, L, P_b, n) \right\}$
-    and $n \leftarrow n + 1$.
```

Comment:
So $n$ is the ID of the next new individual.  I'll move some of the explanation that came afterwards up top to avoid this confusion, hopefully?